server/zanata-war/src/test/java/org/zanata/ZanataRestTest.java

```diff
@@ -79,12 +84,11 @@ public final void prepareRestEasyFramework() {
                     providerInstance);
         }
 
-        InMemoryClientExecutor executor =
-                new InMemoryClientExecutor(dispatcher);
```

Comment:
This whole class need some major refactoring, and abit of research on how to attach the mock dispatcher into the request. For this PR, I will just suppress the warnings and another PR for the actual refactoring.